### ISSUE TITLE
Need to create arbitrary groups of images for TMS mosaic and web browsing

### ISSUE THREAD

**Issue by bgirardot** [hotosm/OpenAerialMap#121]
As DG and others release imagery, I would like to group them by event and provide a TMS layer for the imagery for the event.

I also need to do this as two groups, one for the event pre-event imagery and one tms layer for the event post event imagery.

To do that now I would have to have two user accounts one for pre-event imagery and one for post event imagery so I could share the user wide TMS / web urls.

**Comment by SColchester** [comment]
Yes this would be great, at the moment sharing all imagery associated with an event is not easy, at the moment the workaround is to create a table in OSM Wiki [like this](https://wiki.openstreetmap.org/wiki/Humanitarian_OSM_Team/Open_Mapping_Hub_-_Latin_America_and_The_Caribbean/Activations/2026_Venezuela_Earthquake#Vantor_Open_Satellite_Images) (which is time consuming and unwieldy when there are lots of images), or to use HOT portal [like this](https://portal.hotosm.org/en/plan/1b604573-8b3d-4506-b372-ccd55fa1c700) (which is not very clean because it mixes image with a bunch of other assets like Tasking Manager projects) to 'group' unique OAM image URLs. 

Having a custom OAM URL to imagery which is grouped or tagged to be presented together would be a huge win for this and would make sharing with our partners and other disaster response organisations / data users much easier and more effective.

**Comment by spwoodcock** [comment]
Related to https://github.com/hotosm/openaerialmap/issues/199. I think this would be a STAC field defined in a v0.3.0 spec, such as `event_id`.

Then in the UI, we can filter by `event_id` or add a button to copy a TMS URL containing all images with a matching `event_id`.

**Comment by SColchester** [comment]
That would be fantastic! Maybe also including the 'phase' field [like Vantor use](https://browser.moregeo.it/external/vantor-opendata.s3.amazonaws.com/events/Nepal-Flooding-Aug-2026/B030001100CF1610.json) (with 'pre' or 'post' as values) would be very valuable for example filtering for all post-event Nepal floods images
